versions.tf

```diff
@@ -1,14 +1,14 @@
 terraform {
-  required_version = ">= 0.13.1"
+  required_version = ">= 1.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.17"
+      version = ">= 5.0"
     }
     time = {
       source  = "hashicorp/time"
-      version = ">=0.7.2"
+      version = ">=0.9"
```

Comment:
```suggestion
      version = ">= 0.9"
```

Reply:
updated in https://github.com/terraform-aws-modules/terraform-aws-dms/pull/42/commits/6ef37ea00686d51687526f60f6bb9765bea7ef18